docs/prediction.md

```diff
@@ -40,6 +40,7 @@ sequences:
         id: [CHAIN_ID, CHAIN_ID]    # multiple ids in case of multiple identical entities
         ...
 constraints:
+    - cyclic: CHAIN_ID
```

Comment:
I think it would be preferable to just set cyclic: True on protein entities instead of adding this as a constraint. Could you please make that change?